build.gradle

```diff
@@ -34,6 +33,7 @@ buildscript {
 
 allprojects {
     repositories {
+        maven { url '/home/scli/more-android-sdk/extra/m2repository' }
```

Comment:
What's this about?

Reply:
That is the dependency on my local build. Will remove it shortly.